zpeclient.go

```diff
@@ -300,3 +334,45 @@ func (c *Cache) authorize(principal string, check AthenzAccessCheck) (bool, erro
 	}
 	return false, nil
 }
+
+// parseUpdateTime - read the last update time as string from configmap, store in the cache
+func (c *Cache) parseUpdateTime(configmap interface{}) error {
+	var lastUpdateTime string
+	c.lock.Lock()
+	defer c.lock.Unlock()
```

Comment:
Can we minimize the amount of time we're locking by moving the locks only to when we're setting variables into the cache object? Similar to how we do it here: https://github.com/yahoo/k8s-athenz-webhook/blob/master/zpeclient.go#L260-L262
This might require a little bit of refactoring this function and combining it with `updateCacheStatus`

Reply:
will do.